### ISSUE TITLE
TypeError: Cannot read property 'title' of undefined

### ISSUE THREAD

**Issue by Zorig** [didacte/ember-meta-meta#11]
Hi, I was trying to work ember-meta-meta in ember-fastboot then it gives me this error 
```
TypeError: Cannot read property 'title' of undefined
``` 
In a console error in `vendor.js` file, it triggers error on this part:
```
define('ember-meta-meta/services/meta', ['exports', 'ember'], function (exports, _ember) {
  'use strict';

  var Service = _ember['default'].Service;

  exports['default'] = Service.extend({
    /**
     Store the default document title
      @method storeDefaultTitle
     */
    storeDefaultTitle: function storeDefaultTitle() {
      this.set('defaultTitle', window.document.title);
    },

    /**
     Set metas to be injected
      For exemple :
      ```js
     this.get('meta').update({
          title: 'This Is News Title',
          description: 'This Is News Description',
          'og:image': 'https://exemple.net/latest-news.png'
        });
     ```
      @method update
     @param {Object} attributes
     */
    update: function update(attributes) {
      this.set('attributes', attributes);
    },
```
How to resolve this one?

**Comment by cecemel** [comment]
hi, 
same issue here.
thanks for all the help.

**Comment by j15e** [comment]
Hello, please see this issue : https://github.com/didacte/ember-meta-meta/issues/8

Tldr ; won't fix because it would require a complete rewrite & other plugins exists doing it already.

Look at [ember-cli-meta-tags](https://github.com/ronco/ember-cli-meta-tags) which use [ember-cli-head](https://github.com/ronco/ember-cli-head/blob/master/app/instance-initializers/fastboot/head.js#L2-L13) which do that circumventing.
